assets/js/blocks/active-filters/block.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "woocommerce/active-filters",
 	"version": "1.0.0",
-	"title": "Active Product Filter Controls",
+	"title": "Active Product Filters Controls",
 	"description": "Display the currently active product filters.",
```

Comment:
@tjcafferkey We're using duplicated description for the filter controls block and the wrapper variation. What do you think about removing the description for controls blocks?

Reply:
I think we can remove the description for the control blocks since they only feature in the wrapper blocks (which have the same description). And you cannot add them via the inserter separately.

Reply:
Removed in [ed5417e](https://github.com/woocommerce/woocommerce-blocks/pull/7354/commits/ed5417eb0e5191ed2f1def2a04c9e724fe70856a)